content/spin/v2/triggers.md

```diff
@@ -13,9 +13,70 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/triggers.md"
+
+> Currently, only the HTTP template supports the `spin add` command. If you wish to use `spin add` to build an application with multiple triggers, **including a Redis trigger**, you should begin with the Redis trigger application. Then, `spin add` your HTTP trigger component(s) as demonstrated below. Please note that additional Redis trigger components cannot be added via `spin add` at this time.
+
+Here is a quick example of creating an application with both HTTP and Redis triggers:
+
+<!-- @nocpy -->
+
+```bash
+# Start with a Redis trigger application
+$ spin new -t redis-rust trigger-example
+Description: A multiple trigger example
+Redis address: redis://localhost:6379
+Redis channel: one
+# Change into to the application directory
+$ cd trigger-example 
+# Create HTTP trigger application
+$ spin add -t http-rust http-trigger-example  
+Description: A HTTP trigger example
+HTTP path: /...
+```
+
+The above `spin new` and `spin add` commands will automatically scaffold the following Spin manifest (`spin.toml` file):
+
+<!-- @nocpy -->
+
+```toml
+spin_manifest_version = 2
+
+[application]
+name = "trigger-example"
+version = "0.1.0"
+authors = ["Fermyon Engineering <engineering@fermyon.com>"]
+description = "A multiple trigger example"
+
+[application.trigger.redis]
+address = "redis://localhost:6379"
+
+[[trigger.redis]]
+channel = "one"
+component = "trigger-example"
+
+[component.trigger-example]
+source = "target/wasm32-wasi/release/trigger_example.wasm"
+allowed_outbound_hosts = []
+[component.trigger-example.build]
+command = "cargo build --target wasm32-wasi --release"
+
+[[trigger.http]]
+route = "/..."
+component = "http-trigger-example"
+
+[component.http-trigger-example]
+source = "http-trigger-example/target/wasm32-wasi/release/http_trigger_example.wasm"
+allowed_outbound_hosts = []
+[component.http-trigger-example.build]
+command = "cargo build --target wasm32-wasi --release"
+workdir = "http-trigger-example"
+watch = ["src/**/*.rs", "Cargo.toml"]
+```
+
+> If you're familiar with [Spin 1.x](/spin/manifest-reference-v1#the-trigger-table), note that [Spin 2 uses the term "trigger"](/spin/manifest-reference#the-trigger-table) to refer to each individual route or channel, rather than the trigger type. It's closer to the `[component.trigger]` usage than to the application trigger.
 
-> If you're familiar with Spin 1.x, note that Spin 2 uses the term "trigger" to refer to each individual route or channel, rather than the trigger type. It's closer to the `[component.trigger]` usage than to the application trigger.
+Except for the application's business logic that an application developer will implement, the above multi-trigger application example is operational and can be built and run using `spin build --up` command.
```

Comment:
This paragraph doesn't really seem to be saying anything (other than maybe "we promise you this really works") - I'd suggest removing it, or rephrasing to clarify its intent.

Reply:
Removed this paragraph. ✅